ckanext/twitter/plugin.py

```diff
@@ -78,4 +88,4 @@ def before_map(self, _map):
                      conditions = {
                          'method': ['POST']
                          })
-        return _map
+        return _map
```

Comment:
Python's PEP8 coding standards specify that a newline should be at the end of files.